test_api.py

```diff
@@ -53,11 +51,20 @@ def test_parse_cached_forecasts(file):
         cached_forecast = json.load(forecast_file)
 
     region_forecast = api.parse_forecast(cached_forecast)
+    _test_region_forecast(region_forecast)
+
+def _test_region_forecast(region_forecast):
     assert region_forecast
+    assert region_forecast["preamble"]
+
+    if "synopsis" in region_forecast["preamble"].lower():
+        if not region_forecast["short_synopsis"]:
+            print(region_forecast["preamble"])
+        assert region_forecast["short_synopsis"]
```

Comment:
There are a few sites in the remote/integration tests where case insensitive "synopsis" appears in the preamble but the synopsis isn't getting successfully parsed. Can you see if you can get those to work? (The print line here can be removed once all sites are working, that's just for debugging).

```
pytest --integration
```